### PR TITLE
Add RenderToGraphics methods to Convert Slide article

### PR DESCRIPTION
**Pull request by petrtetenko**
Add RenderToGraphics methods to Convert Slide article

### CODE REVIEW

**Review by EmSeriki — APPROVED** [review]

File: java/developer-guide/presentation-slide/convert-slide/_index.md [hunk 1]
```diff
@@ -3,144 +3,255 @@ title: Convert Slide
...
+try {
+    // Converts the first slide in the presentation to a Bitmap with the specified size
     BufferedImage bmp = pres.getSlides().get_Item(0).getThumbnail(new Dimension(1820, 1040));
-    {
-        // Save the image in JPEG format
-        ImageIO.write(bmp, "PNG", new File("Slide_0.jpg"));
-    }
+	
+	// Saves the image in the JPEG format
+	ImageIO.write(bmp, "PNG", new File("Slide_0.jpg"));
```
Comment (stasmiroshnichenko) [inline java/developer-guide/presentation-slide/convert-slide/_index.md:88]: Inconsistency in output image format. Please choose one from JPEG or PNG.

File: java/developer-guide/presentation-slide/convert-slide/_index.md [hunk 2]
```diff
@@ -3,144 +3,255 @@ title: Convert Slide
...
+		graphics.translate(12, 12);
+
+		// Renders the first slide to Graphics
+		pres.getSlides().get_Item(0).renderToGraphics(new RenderingOptions(), (Graphics2D) graphics, slideSize);
+	} finally {
+		if (graphics != null) graphics.dispose();
+	}
+
+	// Saves the image in the JPEG format
+	ImageIO.write(slideImage, "png", new File("FramedSlide_0.jpg"));
```
Comment (stasmiroshnichenko) [inline java/developer-guide/presentation-slide/convert-slide/_index.md:118]: Inconsistency in the output image format. Please choose one from JPEG or PNG.

File: java/developer-guide/presentation-slide/convert-slide/_index.md [hunk 3]
```diff
@@ -154,15 +265,13 @@ try {
 
         // Convert slide to a Bitmap object
         BufferedImage bmp = pres.getSlides().get_Item(i).getThumbnail(2f, 2f);
-		
-        // Create file name for an image
-        String outputFilePath = outputDir + "Slide_" + i + ".jpg";
 
         // Save the image in PNG format
-        ImageIO.write(bmp, "PNG", new File(outputFilePath));
+        ImageIO.write(bmp, "PNG", new File("Slide_" + i + ".jpg"));
```
Comment (stasmiroshnichenko) [inline java/developer-guide/presentation-slide/convert-slide/_index.md:270]: Inconsistency in the output image format. Please choose one from JPEG or PNG.

**Review by stasmiroshnichenko — CHANGES_REQUESTED** [review]

File: java/developer-guide/presentation-slide/convert-slide/_index.md (hunk 3 above)
Comment (petrtetenko) [reply java/developer-guide/presentation-slide/convert-slide/_index.md]: Done

File: java/developer-guide/presentation-slide/convert-slide/_index.md (hunk 2 above)
Comment (petrtetenko) [reply java/developer-guide/presentation-slide/convert-slide/_index.md]: Done

File: java/developer-guide/presentation-slide/convert-slide/_index.md (hunk 1 above)
Comment (petrtetenko) [reply java/developer-guide/presentation-slide/convert-slide/_index.md]: Done

**Review by stasmiroshnichenko — APPROVED** [review]

**Pull request merged**